scss/config/_abstracts.scss

```diff
@@ -25,13 +25,15 @@
 /// @prop {function | string} <alias> -
 ///   Use `get-function()` to capture a first-class function,
 ///   or use a string to reference existing functions and alias-keys
-tools.$functions: (
+$functions: (
   'darken': meta.get-function('darken'),
   'desaturate': meta.get-function('desaturate'),
   'rgba': meta.get-function('rgba'),
-  'convert': '#convert-units'
+  'convert': '#convert-units',
 );
 
+tools.$functions: $functions;
```

Comment:
@mirisuzanne This seems like an underlying bug in Sassdoc -- I guess it doesn't know that `tools.$functions` is a variable declaration?